### DISCUSSION TITLE
Easiest way to update changes from QGIS on laptop to Qfield

### DISCUSSION THREAD

**Discussion by sigwig** [opengisch/QField#5064] [Q&A]
I am in the office making maps for my team on the land and only I will be updating the maps regularly. 
When I make an update from QGIS and try to package and push to Qfield. The changes are never reflected in my Qfield app on the phone. The whole process seems unnecessarily complicated, am I missing something?

**Comment by SeqLaz** [comment]
The most efficient and straightforward method to synchronize and maintain your project up to date across devices is by utilizing QFieldCloud.

Furthermore, it's important to note that when you import a project, it generates a new copy of the project in the `<drive>:/Android/data/ch.opengis.qfield/` directory. Consequently, modifications made to the imported project do not directly affect the original project stored in `<drive>:/Android/Documents/`.

For more detailed information, you can refer to the documentation provided at: [QField Storage Documentation](https://docs.qfield.org/get-started/storage/)

**Comment by ThomasDV1984** [comment]
You could make a direct connection with a database on a remote server, then no copy is made and all chances are emmediatelly visable in on qfield. A good internet connection is necessary, on the other hand you will be the only one making the chances in the project so it is maybe not a big problem if the connection is not perfect.
https://docs.qfield.org/get-started/tutorials/advanced-setup-qfc/
